src/main/groovy/edu/oregonstate/mist/locations/resources/LocationResource.groovy

```diff
@@ -121,16 +142,30 @@ class LocationResource extends Resource {
     @Path("combined")
     @Produces(MediaType.APPLICATION_JSON)
     @Timed
-    Response combineSources(@Auth AuthenticatedUser authenticatedUser) {
+    Response combineSources() {
         List<List> locationsList = []
```

Comment:
This API produces two files: 1) locations-combined.json and 2) services.json . Both of these files can be used for bulk updates in ElasticSearch. The two files would go into two separate elasticsearch indices: locations and services.